kcidb/__init__.py

```diff
@@ -252,11 +252,26 @@ def query_main():
 def schema_main():
     """Execute the kcidb-schema command-line tool"""
     sys.excepthook = misc.log_and_print_excepthook
-    description = 'kcidb-schema - Output latest I/O JSON schema'
+    description = 'kcidb-schema - Output latest or older I/O JSON schema'
     parser = misc.OutputArgumentParser(description=description)
+    version = io.schema.LATEST
+    parser.add_argument(
+        'schema_version',
+        metavar='Schema Version',
+        type=int,
+        help=f'KCIDB schema version. Default is {version.major}',
+        nargs='?',
+    )
     args = parser.parse_args()
-    misc.json_dump(io.schema.LATEST.json, sys.stdout,
-                   indent=args.indent, seq=args.seq)
+    major = args.schema_version
+    if major is None:
+        major = version.major
```

Comment:
Actually, if you just specify `version.major` as the `default` for the argument you're adding, you won't have to handle it here.